src/main/java/net/fabricmc/loom/configuration/providers/minecraft/SplitMinecraftProvider.java

```diff
@@ -56,6 +56,10 @@ public List<Path> getMinecraftJars() {
 	public void provide() throws Exception {
 		super.provide();
 
+		if (!canMergeJars()) {
+			throw new UnsupportedOperationException("the split jar configuration is not supported at this moment for minecraft versions where the obfuscated client and server jars cannot be merged!");
+		}
+
```

Comment:
Although not wrong I dont think this is needed, as it will bail out where it checks if `serverBundleMetadata` is not null. This was only added in Minecraft 1.17.

Reply:
ah that's my bad, I'll remove this then!